compiler-core/src/language_server/completer.rs

```diff
@@ -570,6 +571,24 @@ where
         // Do not complete direct module values if the user has already started typing a module select.
         // e.x. when the user has typed mymodule.| we know local module values are no longer relevant
         if module_select.is_none() {
+            let cursor = self
+                .src_line_numbers
+                .byte_index(self.cursor_position.line, self.cursor_position.character);
+
+            // Function arguments and variables
+            // Find the function that the cursor is in
+            // and push the arguments and local variables
+            if let Some(fun) = self.module.ast.definitions.iter().find_map(|d| match d {
+                Definition::Function(f) if f.full_location().contains(cursor) => Some(f),
```

Comment:
Will this suggest completions for the values within the header of the function? Could we have a test to verify it does not please 🙏 

```gleam
pub fn main(x, y, | z) {
  Nil
}